### ISSUE TITLE
Typefaces that aren't centered don't work well with MegaType; we need a per-typeface configuarable offset

### ISSUE THREAD

**Issue by tbredin** [StudioThick/megatype#23]
Recently we've noticed that some typefaces that don't have their cap-height centred inside of their line-height are going to be a problem. 

A good example of this is Adobe Caslon Pro, which sits close to the top of the line-height box:  
https://typekit.com/fonts/adobe-caslon-pro  

The "spooky box" is not even above and below this typeface. 

I propose a new key in the font map, alongside `cap-height`. This could be called something like `cap-position` and would default to `0.5` (centred), but could be tweaked to push Megatype calculations up and down inside the "spooky box". 

In theory, `cap-height` and `cap-position` should handle any combination of scale and centering of characters in a typeface.

**Comment by tbredin** [comment]
Technically this has been solved with v1.1 but we haven't had a chance to thoroughly document this yet so the issue will stay open. A new property has been added that will shift type up and down. 

```
$body-typeface: (
    font-family: "'Helvetica Neue', Helvetica, Arial, sans-serif",
    cap-height: 0.69,
    cap-position: 0.59
);
```

`cap-position` is new.  
If unset / default it's `0.5`

**Comment by tbredin** [comment]
Documented in 40b08ea7596eeeb63e695f0873b0c62f519ec64a
